.changeset/odd-buses-occur.md

```diff
@@ -0,0 +1,8 @@
+---
+"@aws-amplify/ui": patch
+"@aws-amplify/ui-react": patch
```

Comment:
Do we also need `@aws-amplify/ui-react-native` since it is a consumer of the ui package?

Reply:
Done, I feel bad for leaving it out 😅